### ISSUE TITLE
Error on adding api-server-authorized-ip-ranges

### ISSUE THREAD

**Issue by jonnylove** [Azure/azure-cli#9189]
## Describe the bug

**Command Name**
`az aks update`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:


super() takes at least 1 argument (0 given)
Traceback (most recent call last):
  File "/usr/lib64/az/lib/python2.7/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/lib64/az/lib/python2.7/site-packages/azure/cli/core/commands/__init__.py", line 445, in execute
    self.commands_loader.load_arguments(command)
  File "/usr/lib64/az/lib/python2.7/site-packages/azure/cli/core/__init__.py", line 268, in load_arguments
    loader.load_arguments(command)  # this adds entries to the argument registries
  File "/home/jlareau/.azure/cliextensions/aks-preview/azext_aks_preview/__init__.py", line 35, in load_arguments
    super().load_arguments(command)
TypeError: super() takes at least 1 argument (0 given)
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks update -g {} -n {} --api-server-authorized-ip-ranges {}`

## Environment Summary
```
Linux-5.0.8-200.fc29.x86_64-x86_64-with-fedora-29-Twenty_Nine
Python 2.7.15
Shell: bash

azure-cli 2.0.63

Extensions:
aks-preview 0.3.1

```

**Comment by adewaleo** [comment]
@rjtsdl @JackQuincy @mboersma this should be an easy fix.

**Comment by feiskyer** [comment]
the aks-preview extension requires Python 3

**Comment by Karishma-Tiwari-MSFT** [comment]
Is there an update on this issue?

**Comment by adewaleo** [comment]
@feisker are you saying this is an issue for those using python 2 and not 3? Please try to follow up with @Karishma-Tiwari-MSFT and  @jonnylove

**Comment by feiskyer** [comment]
@adewaleo Right, the aks-preview extension only works with Python 3. 

@Karishma-Tiwari-MSFT and @jonnylove Could you switch to Python 3 or use Cloud Shell?

**Comment by Karishma-Tiwari-MSFT** [comment]
Thanks @feiskyer

**Comment by larryms** [comment]
Hmm, I am getting the error in my WSL session even though I believe the cli is calling python3:

```
larry@LACLAMA-SP4:  $ az aks update  --resource-group aksrg --name myAKSCluster  --api-server-authorized-ip-ranges 172.0.0.10/16,168.10.0.10/18
--api-server-authorized-ip-ranges should be list of IPv4 addresses or CIDRs

larry@LACLAMA-SP4: $  az --version
azure-cli                         2.0.70

command-modules-nspkg               2.0.3
core                              2.0.70
nspkg                              3.0.4
telemetry                          1.0.3

Extensions:
aks-preview                        0.4.5
azure-devops                      0.10.0
azure-firewall                     0.1.3

Python location '/opt/az/bin/python3'
Extensions directory '/home/larry/.azure/cliextensions'

Python (Linux) 3.6.5 (default, Jul 25 2019, 08:54:21)
[GCC 7.4.0]

Legal docs and information: aka.ms/AzureCliLegal


Your CLI is up-to-date.
```

**Comment by brendandburns** [comment]
fwiw, this was fixed in #636 also. 

@tjprescott this issue can be closed.
